gh-890: Use git switch to switch to branch (#905)
python/devguide: gitbootcamp.rst

>    # create a new branch from main, without checking it out
>    git branch <branch-name> main
> 
>    # check out the branch
>    git switch <branch-name>
>    # or using old command
>    git checkout <branch-name>

Reviewer: ```suggestion
   # create a new branch from main
   git branch <branch-name> main
   # switch to the new branch
   git switch <branch-name>
```

To switch to a different branch::

   git switch <another-branch-name>

>    # or using old command
>    git checkout <another-branch-name>

Reviewer: ```suggestion
```
